iib/workers/tasks/build.py

```diff
@@ -562,15 +581,15 @@ def _update_index_image_build_state(request_id, prebuild_info):
         image.
     """
     arches_str = ', '.join(sorted(prebuild_info['arches']))
-    payload = {
+    payload: Dict[str, Union[str, Dict[str, List[str]]]] = {
```

Comment:
Shouldn't this be `Dict[str, str]`?

Reply:
That´s right, thanks.

Reply:
Actually, It was alright. 
The second Dict[str,List[str]] is needed for bundle_mapping assigned into the payload variable later (two lines after those selected).

Reply:
What is assigned into the payload['bundle_mapping'] in the tests looks like this:
`{'some-bundle': ['quay.io/some-bundle:v1'], 'some-bundle2': ['quay.io/some-bundle2:v1']}`